### DISCUSSION TITLE
GitHub sends email notifications but not web (or mobile)?

### DISCUSSION THREAD

**Discussion by rajdeep-biswas** [community/community#23752] [New to GitHub]
<p>I have double checked that I have both email and web (and mobile) notifications enabled on my settings page. I came across (this issue)[<a href="https://github.community/t/github-started-clearing-my-web-notifications/279" class="inline-onebox">GitHub started clearing my web notifications</a>] and the solution mentions that if the web client loads up the image, it gets marked as read; when I click on the bell icon on the web, it does show me the notifications listed however, there’s no bell notification marker otherwise that tells me that I have new notifications. I downloaded the mobile app to make sure that the behavior is not just with the web client. Turns out even the mobile app wasn’t sending android notifications. But again, manually opening the notifications panel would have them listed. The only notification I get is from the gmail android app (or when I’m on web, check gmail from time to time). Is it because the android gmail app is loading up the email (before sending me a notification)?</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>Just to be clear, are you saying that your web notifications are being marked as read even if you never open the email on either the web or your phone?</p>
<p>If one of your email clients is pre-loading the 1x1 pixel tracking image before you even open your emails, then that would explain what’s happening, yes! I’d recommend checking all the mail clients you  use, and set them to not load images from <a href="mailto:notifications@github.com">notifications@github.com</a>.</p>

**Reply by enchained** [reply]
I have the same issue (notifications marking as read even if I didn't open any emails). I tried to trigger inbox fetch in all my mail apps, but everything was unread until I opened a specific email. Also I didn't want to disable image load, since most clients apply it for everything and don't allow for domain filtering.

Then I discovered that iOS 15 introduced the ["Protect Mail Activity" feature](https://www.macrumors.com/2021/06/10/ios-15-mail-privacy-protection-tracking-pixels/). It works in the background and located in Settings - Mail - Privacy Protection. Since I use the default mail app amongst others, I disabled this feature (Protect Mail Activity - Off, then Hide IP Address - On and Block All Remote content - Off), waited for a week, and my notifications were not read since then (unless I open them in email by myself).

It was enabled by default on iPhone 13 coming with iOS 15. [Looks like Mac OS is affected too](https://support.apple.com/guide/mail/use-mail-privacy-protection-mlhl03be2866/mac).
I also found a [Mac-specific report of what sounds like the same issue](https://webcache.googleusercontent.com/search?q=cache:bnpJosvQ9TIJ:https://github.community/t/macos-mail-auto-reading-github-notifications/257071+&cd=15&hl=en&ct=clnk). Not sure why the issue isn't widely known, maybe nobody here is using the default mail app or maybe most check their email or GitHub inboxes daily, unlike me?

There is no way to disable this feature just for the github.com domain. Should it be considered a bug since the only solution (besides stopping using the mail app) makes us choose between user privacy and basic notification usability?
